server/test/client.jl

```diff
@@ -0,0 +1,233 @@
+# start Wflow ZMQ server (@async)
+@async begin
+    WflowServer.start()
+end
+
+# Connecting to the Wflow ZMQ Server
+context = Context()
+socket = Socket(context, REQ)
+ZMQ.connect(socket, "tcp://localhost:5555")
+
+function request(message)
+    ZMQ.send(socket, JSON3.write(message))
+    ret_value = JSON3.read(ZMQ.recv(socket), Dict)
+    return ret_value
+end
+
+@testset "Client requests" begin
+
+    @testset "initialization and time functions" begin
+        msg = Dict(
+            "fn" => "initialize",
+            "config_file" => joinpath(@__DIR__, "../../test/sbm_config.toml"),
+        )
+        @test request(msg) == Dict("status" => "OK")
```

Comment:
Just wondering, did you consider working with NamedTuples instead of these small Dicts? It would look nicer with NamedTuples at least.

Reply:
Good suggestion! I have changed the Dicts to NamedTuples.